### PR TITLE
support deploying frontend by k8s

### PR DESCRIPTION
**Pull request by koshuang**
#95

### CODE REVIEW

File: tools/infrastructure/container-based/ochestration/kubernetes/bases/frontend/deployment-app.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - next-starter
+                topologyKey: kubernetes.io/hostname
+              weight: 100
+
+      containers:
+        - name: neo
```
Comment (rt-cff) [inline tools/infrastructure/container-based/ochestration/kubernetes/bases/frontend/deployment-app.yaml:38]: change container name?

**Review by rt-cff — COMMENTED** [review]
r+ with comments

**Comment by koshuang** [comment]
@rt-cff it's just a temporary solution to put sensitive data into git since we have limited time. Ideally we should use Kubernete's secret or use https://github.com/bitnami-labs/sealed-secrets.

File: tools/infrastructure/container-based/ochestration/kubernetes/bases/frontend/deployment-app.yaml (hunk 1 above)
Comment (koshuang) [reply tools/infrastructure/container-based/ochestration/kubernetes/bases/frontend/deployment-app.yaml:38]: Good catch, thanks!

**Comment by rt-cff** [comment]
> @rt-cff it's just a temporary solution to put sensitive data into git since we have limited time. Ideally we should use Kubernete's secret or use https://github.com/bitnami-labs/sealed-secrets.

noted

**Pull request merged**